code/modules/clothing/head/misc_special.dm

```diff
@@ -213,3 +214,30 @@
 	icon_state = "foilhat"
 	item_state = "paper"
 	siemens_coefficient = 2
+
+/obj/item/clothing/head/celtic
+	name = "\improper Celtic crown"
+	desc = "According to legend, Celtic kings would use crowns like this one to shield their subjects from harsh winters back on Earth."
+	icon_state = "celtic_crown"
+	wizard_garb = 1
+
+/obj/item/clothing/head/celtic/equipped(M as mob, head)
```

Comment:
Remove `as mob` and cast the M to `/mob/living/carbon/human` directly, so you can remove the cast to `H` below and make the `istype` a simple `istype(M)`